proposals/route-blinding.md

```diff
@@ -0,0 +1,285 @@
+  * [Dummy hops](#dummy-hops)
+  * [Wallets and unannounced channels](#wallets-and-unannounced-channels)
+  * [Blinded trampoline route](#blinded-trampoline-route)
+* [FAQ](#faq)
+  * [Why not use rendezvous](#why-not-use-rendezvous)
+  * [Why not use HORNET](#why-not-use-hornet)
+
+## Proposal
+
+### Introduction
+
+Route blinding is a lightweight technique to provide recipient anonymity by blinding an arbitrary
+amount of hops at the end of an onion path. It's more flexible than rendezvous because it lets
+senders arbitrarily update amounts and lock times, and reuse a blinded route multiple times (which
+is useful when retrying a failed route or using multi-part payments).
+
+The downside compared to rendezvous is that senders have more leeway to probe by changing various
+variables, so the scheme needs to explicitly defend against probing attacks and may be less private.
+
+Some use-cases where route blinding is useful include:
+
+* Recipient anonymity when receiving payments
+* Using unannounced channels in invoices without revealing them
+* Forcing a payment to go through a specific set of intermediaries that can witness the payment
+* Providing anonymous reply paths for onion messages (offers, stuckless payments, etc)
+
+### Overview
+
+At a high level, route blinding works by having the recipient choose an introduction point and a
+route to himself from that introduction point. The recipient then blinds each node and channel
+along that route with ECDH. The recipient includes the blinded route and a secret in the invoice,
+which allows each node in the blinded route to incrementally unblind the payloads.
+
+This scheme requires all the nodes in the blinded route and the sender to activate support for the
+feature. It only becomes effective once a big enough share of the network supports it.
+
+### Notations
+
+* A node `N(i)`'s `node_id` is defined as: `P(i) = k(i) * G`.
+* Blinded `node_id`s are defined as: `B(i) = b(i) * G`.
+* Ephemeral public keys are defined as: `E(i) = e(i) * G`.
+
+### Requirements
+
+A node `N(r)` wants to provide a blinded route `N(r) <- ... <- N(1) <- N(0)` that must be used
+to receive onion messages.
+
+* The channels used along that route may be either announced or unannounced.
+* When used for payments, intermediate nodes in the blinded route MUST NOT learn `payment_secret`.
+* Intermediate nodes in the blinded route MUST NOT learn the `node_id` or `scid` of other
+  intermediate nodes except for their immediate predecessor or successor.
+* Intermediate nodes in the blinded route MUST NOT learn their distance to the recipient `N(r)`.
+* Senders MUST NOT learn the real `node_id`s and `scid`s of the blinded intermediate hops after the
+  introduction point `N(0)`.
+* If `N(r)` creates multiple blinded routes to herself, senders MUST NOT be able to tell that these
+  routes lead to the same recipient (unless this information is leaked by higher layers of the
+  protocol, such as using the same `payment_hash`).
```

Comment:
I don't understand your point...recipients are building blinded routes, this bullet point simply means that wallet implementations must be careful to not create too many distinct blinded routes for the same `payment_hash`, otherwise it's leaking information to the sender.